src/pages/governance/_components/VotingResult.tsx

```diff
@@ -471,16 +450,34 @@ function LabelWithInfoTooltipAndChecks({
       />
 
       <div className="ml-[5.33px]">
-        <CircularCheckIcon
-          width={14}
-          height={14}
-          className={classNames(
-            "",
-            value.isGreaterThan(BigNumber(comparatorValue))
-              ? "fill-green-600 dark:fill-dark-green-500"
-              : "fill-gray-300 dark:fill-dark-gray-400"
-          )}
-        />
+        {proposalStatus === GovernanceProposalStatus.VOTING ? (
+          <CircularCheckIcon
+            width={14}
+            height={14}
+            className={classNames(
+              "",
+              baseValue!.isGreaterThanOrEqualTo(BigNumber(comparatorValue))
```

Comment:
```suggestion
              baseValue!.isGreaterThanOrEqualTo(comparatorValue)
```

Reply:
removed all BigNumber to use new BigNumber